man/systemd-sysusers.xml

```diff
@@ -94,6 +97,22 @@
         paths. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--target=<replaceable>PATH</replaceable></option></term>
+        <listitem><para>When this option is given, one ore more positional arguments
+        must be specified. All configuration files found in the directories listed in
+        <citerefentry><refentrytitle>sysusers.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        will be read, and the configuration given on the command line will be
+        handled instead of and with the same priority as the configuration file
+        <replaceable>PATH</replaceable>.</para>
+
+        <para>This option is intended to be used when package installation scripts
+        are running and files belonging to that package are not yet available on
+        disk, so their contents must be given on the command line, but the admin
+        configuration might already exist and should be given higher priority.
+        </para></listitem>
```

Comment:
Could you provide examples? Otherwise, this logic is hard to understand for me without looking the code.